templates/cases/case_read.haml

```diff
@@ -163,7 +163,10 @@
             .container-fluid
               .row
                 .contact-field-label.col-sm-6
-                  - trans "Name"
+                  - if contact_display_format == 'urn'
+                    - trans "URN"
```

Comment:
I'm happy to change the display to use the scheme rather than `URN` but changing the display to the example you've given requires a rework of the code since `get_contact_display` is only expected to return a string and really shouldn't be returning a list of urns.